src/test_typing.py

```diff
@@ -1564,7 +1564,7 @@ def foo(a: 'Noode[T]'):
     def test_no_type_check(self):
 
         @no_type_check
-        def foo(a: 'whatevers') -> {}:
+        def foo(a: 'int') -> {}:
```

Comment:
Hum, I thought it was intentional that this test references something that doesn't exist. If a linter complains it is technically in violation of PEP 484 (by not understanding the `@no_type_check` decorator), but the pragmatic is to add yet another `# noqa`.

Reply:
Indeed, I will switch these to `# noqa` now.